api/enki.py

```diff
@@ -70,10 +70,13 @@ class EnkiAPI(BaseCirculationAPI, HasSelfTests):
 
     DESCRIPTION = _("Integrate an Enki collection.")
     SETTINGS = [
-        { "key": Collection.EXTERNAL_ACCOUNT_ID_KEY, "label": _("Library ID"), "required": True },
         { "key": ExternalIntegration.URL, "label": _("URL"), "default": PRODUCTION_BASE_URL, "required": True, "format": "url" },
     ] + BaseCirculationAPI.SETTINGS
 
+    LIBRARY_SETTINGS = [
+        { "key": Collection.EXTERNAL_ACCOUNT_ID_KEY, "label": _("Library ID"), "required": True },
```

Comment:
You can technically call a key-value pair whatever you want, but this is the name of a field in the `collections` table and also using it as the name of a key-value pair is confusing. I'd call it something custom and very specific, like "enki_library_id", just because we've got so many IDs floating around.